patches/chrome-browser-resources-settings-site_settings-site_list_entry.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/settings/site_settings/site_list_entry.html b/chrome/browser/resources/settings/site_settings/site_list_entry.html
+index 13665a495c7e22ae0b068e1ca80608c205e72ac1..73fc8f1cbe94361c74cf505cc02af756fb0eb6b9 100644
+--- a/chrome/browser/resources/settings/site_settings/site_list_entry.html
++++ b/chrome/browser/resources/settings/site_settings/site_list_entry.html
+@@ -55,7 +55,7 @@
+       <template is="dom-if" if="[[model.incognito]]">
+         <cr-tooltip-icon id="incognitoTooltip"
+             icon-aria-label="$i18n{incognitoSiteExceptionDesc}"
+-            icon-class="settings20:incognito"
```

Comment:
@petemill can this be done with //brave/browser/resources/settings/brave_overrides/icons.js override instead of patching?